data/json/furniture.json

```diff
@@ -2988,6 +2988,57 @@
             ]
         }
     },
+    {
+        "type" : "furniture",
+        "id" : "f_butcher_rack",
+        "name": "butchering rack",
+        "description": "Butchering rack designed to hang the carcass in the air.",
+        "symbol": "^",
+        "bgcolor": "brown",
+        "move_cost_mod": 2,
+        "required_str": -1,
+        "flags": ["PLACE_ITEM", "TRANSPARENT", "FLAMABLE_ASH", "MOUNTABLE", "ALLOW_FIELD_EFFECT", "EASY_DECONSTRUCT", "BUTCHER_EQ", "ORGANIC"],
+        "deconstruct": {
+            "items": [
+            { "item": "stick_long", "count": 6 }
+            ]
+        },
+        "bash": {
+            "str_min": 8, "str_max": 30,
+            "sound": "crunch!",
+            "sound_fail": "whump!",
+            "items": [
+                { "item": "stick_long", "count": [1, 6] }
+            ]
+        }
+    },
+    {
+        "type" : "furniture",
+        "id" : "f_metal_butcher_rack",
+        "name": "metal butchering rack",
+        "description": "Metal butchering rack designed to hang the carcass in the air.  It can be deconstructed and folded for easy transportation.",
+        "symbol": "^",
+        "bgcolor": "light_gray",
+        "move_cost_mod": 2,
+        "required_str": -1,
+        "deployed_item": "metal_butcher_rack",
+        "examine_action": "deployed_furniture",
+        "flags": ["PLACE_ITEM", "TRANSPARENT", "MOUNTABLE", "ALLOW_FIELD_EFFECT", "BUTCHER_EQ"],
+        "deconstruct": {
```

Comment:
Why use both the construction system and furniture deployment for this? Would be simpler to pick one.

Reply:
I agree, but which one would be better? I couldn't decide so I left both for testing, before removing one of them. So far deployment method is slightly more annoying becouse it pops up a query if you want to disassemble the item every time you want to check the tile for butchery results/yields.On the other side I believe that bloating construction menu with simple tasks is not a good thing either.

Reply:
I'd go with furniture deployment. It's simpler and more intuitive than using the construction menu and it's a very minor inconvenience when it queries for disassembly.

Reply:
I actually like the idea of both; one for early butcherers and a more advanced "travel" type. However the weight/volume might need to be adjusted.